src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/roi_align/roi_align_kernel_ref.h

```diff
@@ -7,16 +7,19 @@
 
 namespace kernel_selector {
 
+enum class roi_aligned_mode : uint32_t { ASYMMETRIC, HALF_PIXEL_FOR_NN, HALF_PIXEL };
+
 struct roi_align_params : public base_params {
     roi_align_params() : base_params{KernelType::ROI_ALIGN} {}
 
     int sampling_ratio = 0;
     float spatial_scale = 1.f;
-    PoolType mode = PoolType::MAX;
+    PoolType pooling_mode = PoolType::MAX;
+    roi_aligned_mode aligned_mode = roi_aligned_mode::ASYMMETRIC;
 
     ParamsKey GetParamsKey() const override {
         auto k = base_params::GetParamsKey();
-        k.EnablePoolType(mode);
+        k.EnablePoolType(pooling_mode);
```

Comment:
Do we need to add aligned_model in GetParamsKey()

Reply:
I suppose we don't since aligned mode is an operation specific option.